src/components/VmSnapshots/RestoreConfirmationModal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropsTypes from 'prop-types'
+import { Modal } from 'patternfly-react'
+import { msg } from '../../intl'
+
+const RestoreConfirmationModal = ({ show, onRestore, description, onClose }) => {
+  return (
+    <Modal onHide={onClose} show={show}>
+      <Modal.Header>
+        <button
+          className='close'
+          onClick={onClose}
+        >
+          <span className='pficon pficon-close' title='Close' />
+        </button>
+        <Modal.Title>{msg.confirmRestore()}</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <p dangerouslySetInnerHTML={{ __html: msg.areYouSureYouWantToRestoreSnapshot({ snapshotName: `"<strong>${description}</strong>"` }) }} />
+        <p>{msg.thisOperationCantBeUndone()}</p>
+      </Modal.Body>
+      <Modal.Footer>
+        <button className='btn btn-default' onClick={onClose}>{msg.cancel()}</button>
+        <button className='btn btn-primary' onClick={onRestore}>{msg.restore()}</button>
+      </Modal.Footer>
+    </Modal>
+  )
+}
+
+RestoreConfirmationModal.propTypes = {
+  show: PropsTypes.bool.isRequired,
+  description: PropsTypes.string.isRequired,
```

Comment:
**s/description/snapshotName/** since the value is put directly in an externalized string with a placeholder of that same name.

Reply:
Agree, `description` denotes longer text with an explanation. This prop is used just as a parameter of that.